gradoop-flink/src/main/java/org/gradoop/flink/io/impl/rdbms/connection/FlinkConnect.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.flink.io.impl.rdbms.connection;
+
+import org.apache.flink.api.java.DataSet;
+import org.apache.flink.api.java.ExecutionEnvironment;
+import org.apache.flink.api.java.io.jdbc.JDBCInputFormat;
+import org.apache.flink.api.java.io.jdbc.split.GenericParameterValuesProvider;
+import org.apache.flink.api.java.typeutils.RowTypeInfo;
+import org.apache.flink.types.Row;
+
+/**
+ * Queries the relational data
+ */
+public class FlinkConnect {
```

Comment:
Not sure whether this is a good class name or not. Sounds like we want to connect to flink which is wrong I guess. Please choose a meaningful name.

I somehow dislike the whole thing. The method is called "connect" but actually queries the data and returns a DataSet of Rows. 

Probably something like "query" for the method and RdbmsConnectionHelper for the class?